addons/costume-editor-shortcuts/addon.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Costume Editor Shortcuts",
+  "description": "Enables some keyboard shortcuts in the Costume Editor.",
+  "credits": [
+    {
+      "name": "Hollow_Song's Dad",
+      "link": "https://scratch.mit.edu/users/Hollow_Song/"
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true,
```

Comment:
Could this be removed unless the feature can properly handle being turned off and back on again without a page reload?

Reply:
Removed